server/package.json

```diff
@@ -8,11 +8,16 @@
     "npm": "5.0.3"
   },
   "dependencies": {
-    "express": "^4.15.4"
+    "concurrently": "^3.5.0",
+    "express": "^4.15.4",
+    "mongoose": "^4.12.0"
   },
   "devDependencies": {},
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.js",
+    "server": "node index.js",
+    "client": "npm run start --prefix ../client",
+    "dev": "concurrently \"npm run server\" \"npm run client\""
   },
```

Comment:
"npm run dev" within the server directory will run these two scripts concurrently, which starts up the client and server